spec/automation/unit/method_validation/vmware_validate_request_spec.rb

```diff
@@ -0,0 +1,69 @@
+describe "Auto Approval Request Validation for Infrastructure" do
+  include Spec::Support::QuotaHelper
+  let(:ws) { MiqAeEngine.instantiate("/System/Request/Call_Method?#{method}&#{args}&#{@value}", @user) }
```

Comment:
I really like these tests, but I wish I had an understanding of the underlying values ( not changed since not in the diff ) but I'm finding the error messages are only one sided and I'm not sure what they're proving.